### PR TITLE
Update command-line.mdx for the project spec file name seems not match the subfolder

### PR DESCRIPTION
**Pull request by chengzhangHu**
the project spec file name seems not match the subfolder
image

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@chengzhangHu 

Good catch!

I suggest instead to change the folder structure and replace `integration` with `e2e`.

 `integration` was the default for legacy Cypress
 `e2e` is the default for current Cypress

That would mean that the command line example stays unchanged.

BTW: You will need to sign the CLA (see above) for your PR to be processed.

**Comment by nagash77** [comment]
Hi @chengzhangHu , as @MikeMcC399 you will need to sign the CLA before any PRs can be accepted.  I also think Mike is correct in the suggested approach.  Could you please make those changes and I will get this reviewed!

**Comment by MikeMcC399** [comment]
@chengzhangHu 

Are you going to continue with this PR?

You didn't respond so far to any comments.

**Comment by MikeMcC399** [comment]
- Since there was no further response from the PR submitter @chengzhangHu to the suggested corrections, I submitted a new PR https://github.com/cypress-io/cypress-documentation/pull/5574 to correct the documentation issue.
- This PR is now superseded and can be closed.

**Comment by MikeMcC399** [comment]
@chengzhangHu 

Please close this PR, since the issue has been corrected. See the close button below 🔽 

Thank you for reporting the documentation problem!

**Review by emilyrohrbough — APPROVED** [review]
Nice catch ! Looks like we missed this updated when v10 was released.

File: docs/guides/guides/command-line.mdx [hunk 1]
```diff
@@ -446,7 +446,7 @@ app/
 If we are in the `app` folder, we can run the specs using the following command
 
 ```shell
-cypress run --project tests/e2e --spec ./tests/e2e/cypress/e2e/spec.cy.js
+cypress run --project tests/e2e --spec ./tests/e2e/cypress/integration/spec.js
```
Comment (emilyrohrbough) [inline docs/guides/guides/command-line.mdx:449]:
```suggestion
cypress run --project tests/e2e --spec ./tests/e2e/cypress/integration/spec.cy.js
```

**Comment by emilyrohrbough** [comment]
Closed as @MikeMcC399 fixed this in a different PR.

**Comment by MikeMcC399** [comment]
@emilyrohrbough 

> Closed as @MikeMcC399 fixed this in a different PR.

- Yes, it was fixed in https://github.com/cypress-io/cypress-documentation/pull/5574 (see above https://github.com/cypress-io/cypress-documentation/pull/5467#issuecomment-1851313965).
- Also the submitted never signed the CLA.

**Pull request closed without merging**